gatsby-theme-auth0/src/auth/service.ts

```diff
@@ -11,6 +11,11 @@ export interface SessionState {
   idToken?: string;
 }
 
+export interface LogoutOptions {
+  returnTo?: string;
+  client_id?: string;
```

Comment:
Should this be `clientID`?

Reply:
Actually, instead of defining a new type interface, let's use `auth0.LogoutOptions`.